src/I18n/DateFormatTrait.php

```diff
@@ -135,6 +170,11 @@ public function i18nFormat($format = null, $timezone = null, $locale = null)
     {
         $time = $this;
 
+        // This is required for testI18nFormatWithOffsetTimezone to pass
```

Comment:
I could need a hand with diffing natural e.g. 'America/Vancouver' and numerical e.g. '+09' DateTimeZones.